be/src/olap/delta_writer.cpp

```diff
@@ -231,7 +231,24 @@ int64_t BaseDeltaWriter::mem_consumption(MemType mem) {
     return _memtable_writer->mem_consumption(mem);
 }
 
-void DeltaWriter::_request_slave_tablet_pull_rowset(const PNodeInfo& node_info) {
+Status DeltaWriter::_safe_get_file_size(const std::string& file_path, int64_t* file_size) {
+    CHECK(file_size != nullptr) << "Null output parameter in safe_get_file_size";
+
+    try {
+        if (!std::filesystem::exists(file_path)) {
```

Comment:
There is no need to check if the file exists. If the file does not exist, the file_size function will throw an exception.